content/21.nutraceuticals.md

```diff
@@ -77,36 +77,50 @@ These include n-3 PUFA, zinc, vitamins C and D, and probiotics.
 Currently, Karolinska University Hospital is running a trial that will measure the levels of SPM as a secondary outcome following intravenous supplementation of n-3 PUFA in hospitalized COVID-19 patients to determine whether n-3 PUFA provides therapeutic value [@clinicaltrials:NCT04647604].
+Therefore, while this mechanism provides theoretical support for a role for n-3 PUFAs against COVID-19, experimental support is still needed.
 
-The increased risk of thrombotic complications in COVID-19 infected patients was reported relatively late in comparison to other manifestations of COVID-19 [@doi:10.1016/j.thromres.2020.04.013; @doi:10.1182/blood.2020006000].
+A third possible mechanism by which n-3 PUFAs could benefit COVID-19 patients arises from the fact that some COVID-19 patients, particularly those with comorbidities, are at a significant risk of thrombotic complications including arterial and venous thrombosis [@doi:10.1182/blood.2020006000; @doi:10.1007/s11239-020-02134-3].
+Therefore, the use of prophylactic and therapeutic anticoagulants and antithrombotic agents is under consideration [@doi:10.1016/j.jacc.2020.04.031; @doi:10.3389/fnut.2020.583080].
 Considering that there is significant evidence that n-3 fatty acids and other fish oil-derived lipids possess antithrombotic properties and anti-inflammatory properties [@doi:10.1016/j.prostaglandins.2018.09.005; @doi:10.1016/j.blre.2020.100694], they may have therapeutic value against the prothrombotic complications of COVID-19.
-Based on concern among the medical community that the use of investigational therapeutics on COVID-19 patients already on antiplatelet therapies due to a pre-existing comorbidities may lead to issues with dosing and drug choice, and/or negative drug-drug interactions [@doi:10.1016/j.jacc.2020.04.031], this supplementation holds particular interest for the treatment of patients already receiving pharmaceutical antiplatelet therapies.
-As a result, the use of other therapeutics, such as dietary sources of n-3 fatty acids or nutraceuticals with antiplatelet activities, may be beneficial and warrant further investigation.
+In particular, concerns have been raised within the medical community about using investigational therapeutics on COVID-19 patients who are already on antiplatelet therapies due to pre-existing comorbidities because the introduction of such therapeutics could lead to issues with dosing and drug choice and/or negative drug-drug interactions [@doi:10.1016/j.jacc.2020.04.031].
+In such cases, dietary sources of n-3 fatty acids or other nutraceuticals with antiplatelet activities could hold particular value for reducing the risk of thrombotic complications in patients already receiving pharmaceutical antiplatelet therapies. <!--TO DO: is this being explored/tested?-->
```

Comment:
I would say that this is being explored in the literature, but I haven't seen any evidence of trials looking at platelet activities as a primary or secondary endpoint. Its a labour intensive trial if looking beyond the standard prothrombin time etc.

Reply:
As you see below, they are mainly interested in the anti-inflammatory potential.